src/Nethermind/Nethermind.Evm/EvmStack.cs

```diff
@@ -161,17 +151,14 @@ public void PushUInt32(in int value)
     /// </remarks>
     public void PushUInt256(in UInt256 value)
     {
-        Span<byte> word = _bytes.Slice(Head * WordSize, WordSize);
-        ref byte bytes = ref MemoryMarshal.GetReference(word);
-
+        ref byte bytes = ref PushBytesRef();
         if (Avx2.IsSupported)
         {
             Word shuffle = Vector256.Create(
-                (byte)
-                31, 30, 29, 28, 27, 26, 25, 24,
-                23, 22, 21, 20, 19, 18, 17, 16,
-                15, 14, 13, 12, 11, 10, 9, 8,
-                7, 6, 5, 4, 3, 2, 1, 0);
+                0x18191a1b1c1d1e1ful,
+                0x1011121314151617ul,
+                0x08090a0b0c0d0e0ful,
+                0x0001020304050607ul).AsByte();
```

Comment:
shouldn't this vector be a static or something?

Reply:
Jit automagically handles that for you for vectors 
![image](https://github.com/user-attachments/assets/62b7841d-8872-4023-9631-55c7e36a8caf)